### DISCUSSION TITLE
Make facecam work in PIP

### DISCUSSION THREAD

**Discussion by cslaviero** [blinksh/blink#1661] [Ideas]
Hi, I was wondering if facecam could work in PiP mode while Blink is in background. My usecase (not exclusive) would be that of a professor using Blink to teach programming and computer science stuff, and moving back and forth from drawing in some whiteboard app to Blink. iPadOS16 has enabled developers to keep the webcam enabled while in background (or so) : https://developer.apple.com/documentation/avfoundation/capture_setup/accessing_the_camera_while_multitasking .

**Comment by yury** [comment]
I did initial prototype - looks promising.
Tracking issue #1665
